test/e2e/pages/allocation-criteria.js

```diff
@@ -0,0 +1,53 @@
+import { Selector } from 'testcafe'
+
+import { fillInForm } from '../_helpers'
+
+import Page from './page'
+
+class AllocationCriteria extends Page {
+  constructor() {
+    super()
+    this.url = '/allocation/new/allocation-criteria'
+    this.fields = {
+      prisonerCategory: Selector('#prisoner_category'),
+      sentenceLength: Selector('[name="sentence_length"]'),
+      complexCases: Selector('#complex_cases'),
+      completeInFull: Selector('#complete_in_full'),
+      hasOtherCriteria: Selector('#has_other_criteria'),
+      otherCriteria: Selector('#other_criteria'),
+    }
+  }
+
+  async fill() {
+    const fieldsToFill = [
+      {
+        selector: this.fields.prisonerCategory,
+        value: 'C',
+        type: 'radio',
+      },
+      {
+        selector: this.fields.sentenceLength,
+        type: 'radio',
+      },
+      {
+        selector: this.fields.complexCases,
+        type: 'checkbox',
+      },
+      {
+        selector: this.fields.completeInFull,
+        type: 'radio',
+      },
+      {
+        selector: this.fields.hasOtherCriteria,
+        type: 'radio',
+        value: 'Yes',
+      },
+      {
+        selector: this.fields.otherCriteria,
+        value: 'lorem ipsum',
+      },
+    ]
+    return await fillInForm(fieldsToFill)
```

Comment:
You don't need to `await` on a `return`. You can then also remove the `async` from the function call.